DOCUMENTATION.md

```diff
@@ -104,751 +29,740 @@ import uk.gov.service.notify.SendSmsResponse;
 NotificationClient client = new NotificationClient(apiKey);
 ```
 
-Generate an API key by signing in to [GOV.UK Notify](https://www.notifications.service.gov.uk) and going to the **API integration** page.
+To get an API key, [sign in to GOV.UK Notify](https://www.notifications.service.gov.uk/) and go to the __API integration__ page. You can find more information in the [API keys](/#api-keys) section.
```

Comment:
To be with consistent I would only highlight GOV.UK Notify, rather than sign in to GOV.UK Notify